.github/workflows/int_test.yml

```diff
@@ -84,36 +89,50 @@ jobs:
       CUSTOM_LOCATIONS_OID: ${{ inputs.custom-locations-oid }}
       EXTENSION_SOURCE_DIRECTORY: "./azure-iot-ops-cli-extension"
       K3S_VERSION: "v1.28.5+k3s1"
+      CA_FILE: "test-ca.pem"
+      CA_KEY_FILE: "test-ca-key.pem"
     strategy:
       fail-fast: false
       matrix:
-        feature: [default, mq-insecure, no-syncrules, ca-certs]
-        include:
-          # default / limited options
+        feature: [custom-input, default, mq-insecure, no-syncrules, ca-certs]
+        runtime-args:
+          - ${{ inputs.runtime-init-args != '' }}
+        exclude:
```

Comment:
Talked offline about how this works, but perhaps we should document this in the integration test readme somewhere since it's a bit more convoluted